test/testinput/bufr_ncep_mtiasi.yaml

```
@@ -6,7 +6,7 @@
 observations:
   - obs space:
       name: bufr
-      obsdatain: "./testinput/gdas.t12z.mtiasi.tm00.bufr_d" 
+      obsdatain: "./testinput/gdas.mtiasi.t00z.20220216.bufr"
```

Do you know why the original file doesn't work? Please, notice that your added file (`test/testinput/gdas.mtiasi.t00z.20220216.bufr`) has 779 MB, it's too large. I'd revert it to use the original file or reduce the new one.

```suggestion
      obsdatain: "./testinput/gdas.t12z.mtiasi.tm00.bufr_d" 
```

---

@fabiolrdiniz this is my lack of knowledge showing but does the (*).bufr_d differ from (?).bufr in some way. Why do we have two diferent suffixes for a bufr file ( prep bufr vs. ncep bufr?)